src/types/k8s.go

```diff
@@ -6,9 +6,35 @@ type ZarfState struct {
 	Distro        string       `json:"distro"`
 	Architecture  string       `json:"architecture"`
 	StorageClass  string       `json:"storageClass"`
-	Secret        string       `json:"secret"`
-	NodePort      string       `json:"nodePort"`
 	AgentTLS      GeneratedPKI `json:"agentTLS"`
+
+	GitServer             GitServerInfo         `json:"gitServer"`
+	ContainerRegistryInfo ContainerRegistryInfo `json:"containerRegistryInfo"`
+	LoggingSecret         string                `json:"loggingSecret"`
+}
+
+type GitServerInfo struct {
+	PushUsername string `json:"pushUsername"`
+	PushPassword string `json:"pushPassword"`
+	ReadUsername string `json:"readUsername"`
+	ReadPassword string `json:"readPassword"`
+
+	Address string `json:"address"`
+	Port    int    `json:"port"`
+
+	InternalServer bool `json:"internalServer"`
+}
+
+type ContainerRegistryInfo struct {
```

Comment:
Nit, but we don't have any other concept of different registries types in zarf that I now of, would it make sense to use `RegistryInfo` so it's consistent with how we label registry everywhere else?  A library we use calls its `go-containerregistry`, but we just say `registry` for the most part.